transformer_engine/common/cudnn_utils.h

```diff
@@ -10,7 +10,6 @@
 #include <cudnn.h>
 #include <cudnn_frontend.h>
 #include <cudnn_frontend_utils.h>
-#include <cudnn_graph.h>
```

Comment:
Still not fully sure about this change. @ptrendx Why did we add this include in #1389?

Reply:
We should keep this include. We use the cuDNN frontend here:
https://github.com/NVIDIA/TransformerEngine/blob/bac7dd090a47504d9ded01a8c61a24b035f93a00/transformer_engine/common/cudnn_utils.h#L27
It's good practice to ["include what you use"](https://github.com/include-what-you-use/include-what-you-use/blob/master/docs/WhyIWYU.md) so that we can change the includes in one file without affecting others.

Reply:
Aah ok missed this.